sh/ci-builds

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+set -ex
+
+make
+
+#  only cross-compile if cachix is available
```

Comment:
Why? This is a public cache, so we can still pull things from the cache without this variables. We just need them in order to write to the cache.

Reply:
My bad -- I removed the cachix commands from `.travis.yml` because they're also in `sh/cachix`, but that broke public cache use.